explainer.md

```diff
@@ -259,71 +120,96 @@ to express semantics which are implicit for native elements.
                aria-valuenow="3" aria-valuetext="3"></custom-slider>
 ```
 
-Using AOM, a Custom Element author would be able to write a class declaration like this:
+We propose that authors be allowed to set ARIA properties on `ShadowRoot` nodes
+to express "default" semantics for Web Components:
 
 ```js
 class CustomCheckbox extends HTMLElement {
+
+  // ...
+
   constructor() {
     super();
+    this.attachShadow({mode: 'open'});  // mode may also be "closed".
+
+    // ... any other set-up
+  }
 
-    // Apply a role of "checkbox" via the AOM
-    this.accessibleNode.role = "checkbox";
+  connectedCallback() {
+    // Set the default semantics for the custom element
+    // when it is inserted in the page.
+    this.shadowRoot.role = "checkbox";
```

Comment:
just a clarification question: is this value observable from the element? I suspect the answer to my question is "no, it is not observable from outside". If that's the case, we should probably add a note somewhere explaining that.